### PR TITLE
Pass at markup

### PR DESCRIPTION
**Pull request by bronzehedwick**
Use 4 spaces and remove section comments. The 4 spaces should better
show the indentation, which intrinsically shows the section hierarchy
without the need for comments.

Add section tags around each distinct section.

Change the intro section from `h3` tags to a definition list. An `h3` needs to
follow an `h2` always, and a definition list semantically reflects the "key
value pairs" nature of that structure.

Wrap each letter segment in wrapping markup with classes. I'm skipping
ahead a bit to add some extra HTML I know we'll need when styling.

Add a `small` tag around the copyright notice, as that tag has been
re-utilized for legal text since HTML 5.

Rename some of the IDs to make better sure they won't collide with
anything. Have the in-page links go directly to the appropriate heading;
we don't need a wrapping anchor tag.

Use the details/summary widget for native show/hide functionality for
the expandable menu.

Same treatment to resources, with some notable changes. Change the about
heading tag to `h2`, as there can only be a single `h1` on a page.
Change the resources list again to a definition list, since this data is
perfectly suited to that structure.

TODO add back to top links, and menu icon.

### CODE REVIEW

**Review by aniagaska — CHANGES_REQUESTED** [review]
It looks like in the appeal letter's letter-segment_body is missing and instead is replaced by the annotations text

**Comment by bronzehedwick** [comment]
Hi @aniagaska, I took a look at the code and I'm finding the letter body for the appeal section.

Screen Shot 2020-08-05 at 6 52 09 PM


Am I looking in the wrong place? Are you referring to something else?

Thanks!

**Comment by bronzehedwick** [comment]
I also updated my PR to take a pass at the `resources.html` file.

File: index.html [hunk 1]
```diff
@@ -1,268 +1,314 @@
...
+                <h3 class="letter-segment__heading">Email<br>Subject</h3>
+                <pre class="letter-segment__body">RE: Freedom of Information Law Appeal</pre>
+            </div>
+
+            <div class="letter-segment">
+                <h3 class="letter-segment__heading">Address it to the agency's appeal officer</h3>
+                <p class="letter-segment__body">Appeals Officer:</p>
+            </div>
+
+            <h3>Include a description of the records and when they were requested</h3>
```
Comment (aniagaska) [inline index.html:188]: There's something missing here

**Review by aniagaska — CHANGES_REQUESTED** [review]
We're still missing the letter body where there is space to show that the letter needs to be filled in with details 

>  <p>I hereby appeal the denial of access regarding my request, which was made on </p>

**Comment by aniagaska** [comment]
markup1

I made comments where the body is confused with an annotation, and the body is missing because it had a comment after it writing that it had to have bars after it to show that the content had to be filled in

**Comment by aniagaska** [comment]
> Hi @aniagaska, I took a look at the code and I'm finding the letter body for the appeal section.
> 
> Screen Shot 2020-08-05 at 6 52 09 PM
> 
> Am I looking in the wrong place? Are you referring to something else?
> 
> Thanks!

It's something else! Those look good! I added comments, hopefully its more clear now.

**Review by aniagaska — COMMENTED** [review]
The end of the document serves as a conclusion and should be treated similar to the introduction in terms of styling and markup
`<div class="letter-segment">
                <h3 class="letter-segment__heading">If your appeal is rejected, then you can go to court </h3>
                <p class="letter-segment__body">Judicial review of a final agency denial Article 78 of Civial Practice Law and Rules.</p>
            </div>

            <div class="letter-segment">
                <h3 class="letter-segment__heading">FOIL Attorney fees may be awarded to a member of the public</h3>
                <p class="letter-segment__body"><a href="https://www.dos.ny.gov/coog/news/december2017.html">a mandatory fee is rewarded</a> if the court finds that the agency had no reasonable basis for denying access</p>
            </div>`

**Pull request merged**